src/mods/resource.rs

```diff
@@ -187,19 +189,17 @@ pub(crate) fn generate_resource_insert<P: AsRef<Path>, W: Write>(
     } else {
         0
     };
-    let mime_type = mime_guess::MimeGuess::from_path(&path).first_or_octet_stream();
+    let mime_type = mime_guess::MimeGuess::from_path(path).first_or_octet_stream();
     writeln!(
         f,
-        "{}.insert({:?},n(i!({:?}),{:?},{:?}));",
-        variable_name, &key_path, &abs_path, modified, &mime_type,
+        "{variable_name}.insert({key_path:?},n(i!({abs_path:?}),{modified:?},{mime_type:?}));",
```

Comment:
If we use {qqq} we better set rust-version

Reply:
I ran `cargo msrv` on it, and added. thx.